### PR TITLE
Added some Netgear M4300 models

### PR DESCRIPTION
**Pull request by tlourey**
Hi, 

I have added the following Netgear devices and modules: 

- Netgear M4300-52G-PoE+
- Netgear M4300-28G-PoE+
- Netgear APS550W PSU Module

Assuming tests pass, please review. Thanks.

### CODE REVIEW

**Comment by tlourey** [comment]
Hi Team, can someone review? Many thanks. Tim

File: device-types/Netgear/M4300-28G-PoEp.yaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  - name: 1/0/26
+    type: 10gbase-t
+    enabled: true
+    mgmt_only: false
+    description: 10GBE / Stacking Ports
+  - name: 1/0/27
+    type: 10gbase-x-sfpp
+    enabled: true
+    mgmt_only: false
+  - name: 1/0/
```
Comment (danner26) [inline device-types/Netgear/M4300-28G-PoEp.yaml:178]: Looks like this name might me missing the ending number.

File: device-types/Netgear/M4300-28G-PoEp.yaml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  - name: ServicePort
+    type: 1000base-t
+    enabled: true
+    mgmt_only: true
+    label: ServicePort
+    description: Ethernet Management Service Port
+  - name: vlan 1
+    type: virtual
+    enabled: true
+    mgmt_only: false
```
Comment (danner26) [inline device-types/Netgear/M4300-28G-PoEp.yaml:191]: This definition should be removed, we are really only focusing on physical definitions in the devicetype-library

File: device-types/Netgear/M4300-52G-PoEp.yaml [hunk 3]
```diff
@@ -0,0 +1,341 @@
+---
+manufacturer: Netgear
+model: M4300-52G-PoE+
+slug: netgear-m4300-52g-poep
+part_number: GSM4352PA
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Netgear/M4300-52G-PoEp.yaml:8]: This should be removed, you should only use `subdevice_role` if other devices slot into this one, i.e. `device-bays`

`module-bays` do not count as children

File: device-types/Netgear/M4300-28G-PoEp.yaml [hunk 4]
```diff
@@ -0,0 +1,196 @@
+---
+manufacturer: Netgear
+model: M4300-28G-PoE+
+slug: netgear-m4300-28g-poep
+part_number: GSM4328PA
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Netgear/M4300-28G-PoEp.yaml:8]: This should be removed, you should only use `subdevice_role` if other devices slot into this one, i.e. `device-bays`

`module-bays` do not count as children

File: device-types/Netgear/M4300-52G-PoEp.yaml [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+  - name: ServicePort
+    type: 1000base-t
+    enabled: true
+    mgmt_only: true
+    label: ServicePort
+    description: Ethernet Management Service Port
+  - name: vlan 1
+    type: virtual
+    enabled: true
+    mgmt_only: false
```
Comment (danner26) [inline device-types/Netgear/M4300-52G-PoEp.yaml:336]: This definition should be removed, we are really only focusing on physical definitions in the devicetype-library

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Hello @tlourey can you please resolve the issues, or close this PR? Thank you.

**Comment by tlourey** [comment]
Will do. I'm away right now but should be able to next week.

**Comment by danner26** [comment]
Closing this since it has been incomplete for a long period of time. Feel free to work on it then reopen it again if you would like.

**Pull request closed without merging**